src/braket/experimental/autoqasm/transpiler/transpiler.py

```diff
@@ -139,6 +144,7 @@ def transform_ast(
         # canonicalization creates.
         node = continue_statements.transform(node, ctx)
         node = return_statements.transform(node, ctx)
+        node = comparisons.transform(node, ctx)
```

Comment:
This is how we insert the `visit_Compare` conversion from `converters/comparisons.py`

Reply:
Do you know how sensitive this is to the order of the `transform` calls? I would assume `comparisons` is more closely related to `logical_expressions` and so it might logically go next to that one. The `assignments` one turned out to be very sensitive to the ordering, but probably this new one is not so much.